src/foam/u2/detail/SectionView.js

```diff
@@ -111,15 +121,28 @@ foam.CLASS({
               this.start().addClass('subtitle').add(slot$).end();
             })
             .start(self.Grid)
-              .forEach(section.properties, function(p, index) {
-                this.start(self.GUnit, {columns: p.gridColumns})
-                  .show(p.createVisibilityFor(self.data$, self.controllerMode$).map(mode => mode !== self.DisplayMode.HIDDEN))
-                  .tag(self.SectionedDetailPropertyView, {
-                    prop: p,
-                    data$: self.data$
-                  })
-                .end();
-              })
+              .add(this.slot(function(loadLatch) {
+                var view = this.E();
+                
+                if ( ! loadLatch ) {
```

Comment:
reverse here also

Reply:
okay, so setting it to true was a bug. as the default value of loadLatch would make section which are used elsewhere not only in TabbedDetailView blank.

so no need to reverse here

Reply:
okay, so setting it to true was a bug. as the default value of loadLatch would make section which are used elsewhere not only in TabbedDetailView blank.

Reply:
Shouldn't this be: ' if ( loadLatch ) ' ?

Reply:
if we change `if ( ! loadLatch )` to  `if ( loadLatch )` then on default value which is false we will not display section content